shared/src/main/scala/coop/rchain/catscontrib/Alternative_.scala

```diff
@@ -0,0 +1,39 @@
+package coop.rchain.catscontrib
+
+import cats.{Alternative, Applicative, Bifoldable, Foldable, Monad, MonoidK}
+
+trait Alternative_[F[_]] extends MonoidK[F] {
```

Comment:
Is this type class also lawless?

Reply:
All the `Foo_` typeclasses are merely wrappers around the corresponding `Foo` typeclass from cats - just elliminating the inheritance that leads to ambiguous implicits in presence of (most often) another Applicative instance.

I could (1) test laws for `Alternative_` by providing a test-only Applicative instance that bases on another Applicative instance (mind-blown.gif). I could (2) test the `Foo_` classes by providing a stub `Foo` and asserting results as expected. Or we could (3) agree that testing those is an overkill because they merely delegate all methods to the underlying cats `instance`.

Since (1) is an overkill, let's choose between (2) and (3). My feeling is (3) should be sufficient, but I'm fine doing (2) if you insist :)

Reply:
I guess this would be just "nice to have". (1) sounds alright